src/uct/ib/rc/base/rc_iface.c

```diff
@@ -709,9 +709,9 @@ UCS_CLASS_INIT_FUNC(uct_rc_iface_t, uct_iface_ops_t *tl_ops,
     }
 
     /* Create mempool for pending requests */
-    ucs_assert(init_attr->fc_req_size >= sizeof(uct_rc_pending_req_t));
     ucs_mpool_params_reset(&mp_params);
-    mp_params.elem_size       = init_attr->fc_req_size;
+    mp_params.elem_size       = ucs_max(init_attr->fc_req_size,
```

Comment:
why needed to change?

Reply:
because `fc_req_size == 0` for GGA

Reply:
maybe we don't need this mpool at all for GGA?

Reply:
we need it for pending requests